### PR TITLE
Update

### PR DESCRIPTION
**Pull request by mvdheuvel**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://foundedx-master.dbsima-founded-in-romania-ea32d4ff52c453f97274.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://foundedx-master.dbsima-founded-in-romania-ea32d4ff52c453f97274.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Founded In Romania
+# Founded In X
 
-  * Website for [www.foundedinromania.com](http://foundedinromania.com).
+  * Web CMS for founded-in-X websites, originally forked from [www.foundedinromania.com](http://foundedinromania.com).
```
Comment (dbsima) [inline README.md]: It is not just the CMS. It is the whole website which incorporates the CMS and the typeform API.

Besides that I agree with replacing "Romania" with "X".

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   * Created by [Cloudients](http://www.cloudients.com).
 
 ## Features
```
Comment (dbsima) [inline README.md]: P.S.: Also, we should wait or ask for somebody to make a  review before a merge.

**Comment by otzy007** [comment]
Yeah. The idea of a PR is to ask the others if everything it's ok :tongue: 
One more thing. Let's submit the PRs to https://github.com/FoundedX/founded-in-x and not to dbsima/founded-in-romania.

**Pull request merged**